builds/e2e/e2e.yaml

```diff
@@ -265,4 +265,69 @@ jobs:
     - template: templates/e2e-clear-docker-cached-images.yaml
     - template: templates/e2e-run.yaml
       parameters:
-        test_type: http_proxy
\ No newline at end of file
+        test_type: http_proxy
+
+################################################################################
+  - job: mariner1_amd64
+################################################################################
+    displayName: Mariner 1.0 amd64
+    condition: eq(variables['run.EFLOW.amd64'], 'true')
+    pool:
+      name: $(pool.custom.name)
+      demands:
+        - Agent.OS -equals Linux
+        - Agent.OSArchitecture -equals X64
+        - run-new-e2e-tests -equals true
+        - mariner -equals true
+        - mariner-version -equals 1.0
```

Comment:
Do you know if Mainer1 has 1ES agent we can use?

Reply:
I doubt they do, since mariner 1.0 is going to be deprecated in December

Reply:
I only created images for Mariner 2.0 agents. We could create a Mariner 1.0 agent image, but it doesn't seem worth the effort.

Reply:
I agree, I am planning on removing the mariner 1.0 build in December, since all projects will be based of mariner 2.0 by that point.